### ISSUE TITLE
Plot.sphere crashes if not associated with a projection

### ISSUE THREAD

**Issue by Fil** [observablehq/plot#1123]
```
Plot.sphere().plot()
> TypeError: n.sphere is not a function
```

it's all right, but we might want a better error message

**Comment by mbostock** [comment]
Yes, the crash is fine, but the error message could be improved.

**Comment by mbostock** [comment]
Probably the place to fix this is in d3-geo’s path by implementing the sphere method and throwing an error:

https://github.com/d3/d3-geo/blob/main/src/path/string.js

**Comment by Fil** [comment]
I've fixed it in https://github.com/observablehq/plot/pull/1156/commits/dffc4b61e17754b69abd31f6bc54dcec28120549 by ignoring Plot.sphere if not defined; we could add a warning there, or throw an error (or do nothing a delegate to Plot).

**Comment by LeonxLJX** [comment]
I'd like to take this one (`Plot.sphere errors if not associated with a projection`). I'll look into the root cause and follow up with a PR. (claiming via @LeonxLJX)
